horovod/common/operations.cc

```diff
@@ -1,5 +1,6 @@
 // Copyright 2016 The TensorFlow Authors. All Rights Reserved.
 // Modifications copyright (C) 2019 Uber Technologies, Inc.
+// Modifications copyright (C) 2019 Intel Corporation
```

Comment:
Could you move this new copyright line after NVIDIA's?  New copyrights should be added below existing ones.